godot-git-plugin/src/git_api.h

```diff
@@ -42,14 +53,24 @@ class GitAPI : public EditorVCSInterface {
 	bool _shut_down();
 	void _stage_file(const String p_file_path);
 	void _unstage_file(const String p_file_path);
+	void _discard_file(String p_file_path);
+	Array _get_previous_commits();
+	Array _get_branch_list();
+	bool _checkout_branch(String branch);
```

Comment:
We are also following the Godot convention of parameter naming of p_ and r_. This will be required I guess before merging